doc/nrf/introduction.rst

```diff
@@ -8,20 +8,19 @@ About the |NCS|
    :depth: 2
 
 The |NCS| enables you to develop applications for nRF52, nRF53, and nRF91 Series devices.
-It is a set of open source projects maintained by Nordic Semiconductor, consisting of several Git repositories publicly hosted on `GitHub`_:
+The SDK consists of a combination of Nordic-developed software and open source projects, hosted as `Git_` repositories in the `nrfconnect GitHub organization`_. Some of the more prominent repositories include:
 
 * `sdk-nrf`_ repository - contains applications, samples, libraries, and drivers that are specifically targeted at Nordic Semiconductor devices.
 * `sdk-nrfxlib`_ repository - contains closed-source libraries and modules in binary format.
   See the :doc:`nrfxlib documentation <nrfxlib:README>`.
+* `sdk-zephyr`_ repository - contains a fork of the `Zephyr`_ project, which provides samples, libraries, and drivers for a wide variety of devices, including Nordic Semiconductor devices.
+  See the :doc:`documentation <zephyr:index>` in Nordic Semiconductor’s Zephyr fork.
 * `sdk-mcuboot`_ repository - contains a fork of the `MCUboot`_ project, which provides a secure bootloader application.
   You can find the fork in :file:`bootloader/mcuboot` after obtaining the |NCS| source code.
   See the :doc:`documentation <mcuboot:index-ncs>` in Nordic Semiconductor’s MCUboot fork.
-* `sdk-zephyr`_ repository - contains a fork of the `Zephyr`_ project, which provides samples, libraries, and drivers for a wide variety of devices, including Nordic Semiconductor devices.
-  See the :doc:`documentation <zephyr:index>` in Nordic Semiconductor’s Zephyr fork.
 
-Every nRF Connect SDK release consists of a combination of these repositories at different revisions.
-The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, ``sdk-nrf``.
-It contains the SDK manifest file that enables you to manage the repositories as one code base with the west tool.
+Every nRF Connect SDK release consists of a combination of all repositories at different revisions. See the  :ref:`repos_and_revs` section for an exhaustive list of repositories and their current revisions.
+The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, ``sdk-nrf``, which contains the SDK manifest file that helps manage the repositories as one :ref:`code base <dm_code_base>` with the :ref:`ncs_west_intro` tool.
```

Comment:
This: `of the main (or manifest) repository,` could give the impression that those may be different because the `or`, howabout:
```suggestion
The revision of each of those repositories is determined by the current revision of the main (aka manifest) repository, ``sdk-nrf``, which contains the SDK manifest file that helps manage the repositories as one :ref:`code base <dm_code_base>` with the :ref:`ncs_west_intro` tool.
```

Reply:
How about just simply saying (manifest):
```suggestion
The revision of each of those repositories is determined by the current revision of the main (manifest) repository, ``sdk-nrf``, which contains the SDK manifest file that helps manage the repositories as one :ref:`code base <dm_code_base>` with the :ref:`ncs_west_intro` tool.
```

Reply:
Thanks! I prefer @mia-ko's suggestion to remove the "or" altogether

Reply:
> Thanks! I prefer @mia-ko's suggestion to remove the "or" altogether

perfectly fine with me.